docs/manuals/core/versioning/code_example/main_with_multiple_changes.py

```diff
@@ -0,0 +1,18 @@
+import taipy as tp
+from taipy import Config, Scope
+
+
+def example_algorithm(entry: str):
+    # does nothing!
+    return entry
+
+
+input_cfg = Config.configure_data_node("input", path="input.pkl", scope=Scope.GLOBAL, default_data="a_string")
```

Comment:
What's the point of the underscore in the string value?
It makes it look like it's a symbol or something...

Reply:
Haha, let me fix that

```suggestion
input_cfg = Config.configure_data_node("input", path="input.pkl", scope=Scope.GLOBAL, default_data="A string")
```